.env.testnet.local

```diff
@@ -35,3 +34,7 @@ NEXT_PUBLIC_REWARD_DISTRIBUTOR_ADDRESS=0x83Eac3Abe6AAF4a4AAe8B067888FAD600c4cF6B
 NEXT_PUBLIC_PROXY_DESTINATION=https://dev.rws.app.rootstockcollective.xyz
 NEXT_PUBLIC_ENABLE_CORS_BYPASS=true
 NEXT_PUBLIC_ENABLE_FEATURE_V2_REWARDS=true
+
+# Rif Wallet Services
+NEXT_PUBLIC_RIF_WALLET_SERVICES=http://localhost:3000/cors_bypass
+NEXT_PUBLIC_EVENTS_FROM_BLOCK=5784028
```

Comment:
Where does this number come from?
Neither the [backers manager](https://explorer.testnet.rootstock.io/address/0x41841e316f85933247fc7b51c79b76f22eb239bc) or the [reward distributor](https://explorer.testnet.rootstock.io/address/0x83eac3abe6aaf4a4aae8b067888fad600c4cf6b3) were deployed at this block 🤔